### ISSUE TITLE
The method .order() must contain arguments, when model has column "order"

### ISSUE THREAD

**Issue by Tashows** [thoughtbot/administrate#1272]
When trying to edit a model that has a column attribute named "order", this error appears:
`The method .order() must contain arguments.`

I think the issue comes from this line:
https://github.com/thoughtbot/administrate/blob/d7fed05944273ca9bf8a84c56313c212ff3b4ec9/lib/administrate/page/base.rb#L31

From what I understand, this calls the order method instead of the attribute. Not sure if that should/could be changed to call something else, instead of `public_send`. Maybe `read_attribute`? Would that work? Or maybe it's a bad practice to add columns named "order" in the models? I've been using that as a column name in other apps and never had a problem though.

The workaround I followed for now is setting up a virtual attribute (`model_order`) getter and setter for my model. Leaving this here for discussion and troubleshooting for people who might encounter this.

**Comment by dduqueti** [comment]
@Tashows Any work-around on this? I'm getting the same in another section of the the code related to nested has many library. https://github.com/nickcharlton/administrate-field-nested_has_many

**Comment by dduqueti** [comment]
@Tashows Could you explain how this workaround works? I've tried to do it for an association but couldn't make it.

**Comment by Tashows** [comment]
I created a method called `model_order` inside my model:

    def model_order
      self.read_attribute(:order)
    end
    
    def model_order=(value)
      self.write_attribute(:order, value)
    end

And then in the model's dashboard I registered the attribute `:model_order` as a NumberField and everything worked as expected. In the long run, I renamed all my `:order` attributes (I had attributes with that name in other models as well) into `:position` as it was cleaner and I didn't like overriding the attribute method or including a concern in all these models. Hope that helps.
